contribute/CONTRIBUTING-TO-DEVELOPER-DOC.md

```diff
@@ -47,74 +47,5 @@ You can also help with some existing issues under this category available at [de
 - Once the PR is accepted, close the branch.
 - After the PR is merged the development branch in the forked repository can be deleted.
 
-# Git Cheat Sheet 
-
-Fork the openebs/openebs repository into your account, referred in the below instructions as $user. 
-
-## Setting up Development Environment for OpenEBS projects
-```
-working_dir=$GOPATH/src/github.com/openebs
-mkdir -p $working_dir
-cd $working_dir
-```
-
-Set `user` to match your github profile name:
-
-```
-user={your github profile name}
-```
-
-## The steps mentioned here are w.r.t contribution towards openebs/openebs project
-```
-git clone https://github.com/$user/openebs.git
-cd openebs/
-git remote add upstream https://github.com/openebs/openebs.git
-git remote set-url --push upstream no_push
-git remote -v
-```
-
-## Synchronizing your local master(i.e. your $user/openebs project) with upstream master(i.e. openebs/openebs)
-```
-git checkout master
-git fetch upstream master
-git rebase upstream/master
-git status
-git push origin master
-```
-
-## Create Branch (based on feature or bug name)
-```
-git branch <branch_name>
-git checkout <branch_name>
-git push --set-upstream origin <branch_name>
-```
-
-## Synchronizing (i.e rebase) your local branch with upstream master
-```
-git checkout <branch-name>
-git fetch upstream master
-git rebase upstream/master
-git status
-git push
-```
-
-## Make changes to your local branch
-```
-# make your changes
-# keep fetching the commits from upstream master & rebase them here
-git commit
-git push
-
-# submit the PR to upstream from browser link https://github.com/$user/openebs
-```
-
-## Delete Branch
-```
-git push origin --delete <branch_name>
-git branch -d <branch_name>
-```
-
-## Writing a Unit Test
-Though it is important to write unit tests, do not try to achive 100% code coverage if it complicates writing these tests. If a unit test is simple to write & understand, most probably it will be extended when new code gets added. However, reverse will lead to its removal on the whole. In other words, complicated unit tests will lead to decrease in the overall coverage in the long run.
```

Comment:
achive  = achieve
& = and
in the overall = in overall

Reply:
That section has already been changed, a review has to be casted on the changed category. That's the right side.